samples/javascript_nodejs/14.nlp-with-dispatch/home-automation-state.js

```diff
@@ -4,74 +4,76 @@ const HOME_ATUOMATION_STATE_PROPERTY = 'homeAutomation.state';
 
 class DeviceState {
     /**
-     * 
+     *
      * @param {String} Device Name
-     * @param {String} Room the device is in 
-     * @param {String} Device properties - e.g. temperature=72'F, brightnessLevel=20% 
+     * @param {String} Room the device is in
+     * @param {String} Device properties - e.g. temperature=72'F, brightnessLevel=20%
      * @param {String} State of the device - on/ off/ standby/ pause etc.
      */
-    constructor(device, room, deviceProperty, deviceState) { 
-        this.deviceName = device ? device : '';
-        this.room = room ? room : '';
-        this.deviceProperty = deviceProperty ? deviceProperty : '';
-        this.deviceState = deviceState ? deviceState : '';
+    constructor(device, room, deviceProperty, deviceState) {
+        this.deviceName = device || '';
+        this.room = room || '';
+        this.deviceProperty = deviceProperty || '';
+        this.deviceState = deviceState || '';
     }
 };
 
 class HomeAutomationState {
     /**
-     * 
-     * @param {object} convoState Conversation state
-     * @param {object} userState User state 
+     *
+     * @param {ConversationState} convoState Conversation state
```

Comment:
in the other samples we use 'conversationState' as the variable name, not 'convoState'.  That would require a change here and within this function